styles/main.scss

```diff
@@ -66,6 +69,10 @@ $font-family-secondary: "adelle", "Aleo", "serif";
   color: $secondary;
 }
 
+.aleo {
```

Comment:
Not a fan of doing it this way, I feel like someone who looks at your element later on will have no idea what aleo is?
Since we know from the design system library that main headings and quote body text are meant to be using this font, I think something more along the lines of 
```
.main-heading {
   font-family: $font-family-secondary;
   ... other styling
}

p.quote-body-text {
   font-family: $font-family-secondary;
   ... other styling
}
```
Even though it's 'repeated' twice here, I think it'll better communicate the idea that this is meant to be a core design decision that should apply to all main headings or to all quote body text?

Reply:
Also, within the main.scss please use variables for these kind of situations, esp colors